contents/docs/libraries/next-js.md

```diff
@@ -279,7 +280,7 @@ PostHog is now set up and ready to go. Files and components accessing PostHog on
 
 #### Pageleave events (optional)
 
-To send `$pageleave` events after setting `capture_pageview` to false, you can set the `capture_pageleave` option to `true`.
+To capture pageleave events, first, we need to set `capture_pageleave: true` in the initialization because setting `capture_pageview: false` disables it.
```

Comment:
```suggestion
To capture pageleave events, we need to set `capture_pageleave: true` in the initialization because setting `capture_pageview: false` disables it.
```